src/testing/marks.zig

```diff
@@ -8,27 +6,35 @@
 //! benefits are:
 //! - tests are more resilient to refactors
 //! - production code is more readable (you can immediately jump to a specific test)
+//!
+//! At the surface level, this resembles usual code coverage, but the idea is closer to traceability
+//! from safety-critical systems:
+//!     <https://en.wikipedia.org/wiki/Requirements_traceability>
```

Comment:
Hat tip to @jamesmunns for teaching me about traceability back at Ferrous!

Reply:
🫡